### PR TITLE
Add Arzu Caner's info to the collaborator's section

### PR DESCRIPTION
**Pull request by arzucaner**
## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Documentation
- [ ] Other

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [ ] I have tested my changes locally.
- [ ] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

**Comment by mads6879** [comment]
Can we pls have a screenshot since the collaborators.yml and the image seems fine but it wud be good see if these are rendered properly? Thanks.

File: _data/collaborators.yml [hunk 1]
```diff
@@ -93,3 +93,15 @@
       link: https://www.linkedin.com/in/eleonora-belova-7069baa5/
     - name: github
       link: https://github.com/nora-weisser
+
+- name: Arzu Caner
+  position: Software Developer
+  comment: Website Collaborator
```
Comment (dricazenck) [inline _data/collaborators.yml:99]: I think you can be moved to core-team instead of collaborator, but it is up to you.

Comment (arzucaner) [reply _data/collaborators.yml:99]: Thank you for your comment @dricazenck  I would like to be on the core team but I'm not sure what to write there. Do you have any suggestions regarding this?

Comment (dricazenck) [reply _data/collaborators.yml:99]: I think you can re-place Marina's position: `Communication and Networking`

**Review by mads6879 — COMMENTED** [review]
I suppose PR #297 supersedes this one as everybody has been rearranged on the team's page.

**Comment by khairahscorner** [comment]
Closed with #297

**Pull request closed without merging**